torch/testing/_internal/distributed/rpc/rpc_test.py

```diff
@@ -3635,6 +3635,35 @@ def test_cannot_infer_backend_from_options(self):
                 rpc_backend_options=rpc_backend_options,
             )
 
+    @dist_init
+    def test_local_rref_backward(self):
+        t = torch.rand(10, 10, requires_grad=True)
+        rref = rpc.RRef(t.sum())
+        rref.backward()
+        self.assertEqual(torch.ones_like(t), t.grad)
+
+        with dist_autograd.context() as context_id:
```

Comment:
shall we add a test where part of the distributed autograd graph is on a different rank?